src/components/Nav.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+// import userIcon from '../Assets/user-regular.svg';
+import logo from '../Assets/logo.svg';
```

Comment:
- [ ] You have done great installing axios... But as I pointed out in the previous review, the error with the assets import while starting the app is still there.. I'll suggest you check the import path correctly. 👍 Also I'll suggest you actually run and test your project properly to make sure it works as expected before requesting for review.. 💯 😄